modules/tensor_mechanics/src/materials/ComputeReturnMappingStress.C

```diff
@@ -6,17 +6,17 @@
 /****************************************************************/
 #include "ComputeReturnMappingStress.h"
 
-#include "RecomputeRadialReturn.h"
+#include "StressUpdateBase.h"
 
 template<>
 InputParameters validParams<ComputeReturnMappingStress>()
 {
   InputParameters params = validParams<ComputeFiniteStrainElasticStress>();
   params.addClassDescription("Compute stress using a radial return mapping implementation for creep or creep combined with plasticity");
-  params.addParam<unsigned int>("max_iterations", 30, "Maximum number of newton iterations in the radial return material");
-  params.addParam<Real>("relative_tolerance", 1e-5, "Relative convergence tolerance for the newton iteration within the radial return material");
-  params.addParam<Real>("absolute_tolerance", 1e-5, "Absolute convergence tolerance for newton iteration within the radial return material");
-  params.addParam<bool>("output_iteration_info", false, "Set true to output newton iteration information from the radial return material");
+  params.addParam<unsigned int>("max_iterations", 30, "Maximum number of picard iterations over the stress change after all update materials are called");
```

Comment:
I guess this is technically correct, but we might want to reword this a bit to avoid any potential confusion with multiapp picard iterations.